src/main/java/com/microsoft/azure/functions/annotation/ExponentialBackoffRetry.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.functions.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>
+ * Defines an exponential backoff retry strategy, where the delay between retries
+ * will get progressively larger, limited by the max/min specified.</p>
+ *
+ * @since 1.0.0
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface ExponentialBackoffRetry {
+    /**
+     * The maximum number of retries that will be attempted.
+     * @return The maximum retry count.
+     */
```

Comment:
Same here, Introduce Enum RetryStrategy like you  had in previous commit. Set the RetryStrategy to "ExponentialBackoff" --> this mean mvn plug can convert information in the annotation directly to function.json  --> generic implementation

Reply:
Done